src/main/webapp/index.html

```diff
@@ -1,29 +1,167 @@
-    <title>TEAM-23 PROJECT</title>
-    <link rel="stylesheet" href="style.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <header>
-      <div class="navbar">
-        <a href="#">TITLE</a>
-        <a href="#">HOME</a>
-        <a href="#">LINK2</a>
-        <a href="#">LINK3</a>
-        <div class="search-container">
-          <form action="#">
-            <input type="text" placeholder="Search.." name="search" />
-            <button type="submit"><i class="fa fa-search"></i></button>
-          </form>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link href="styles.css" rel="stylesheet"> -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" >
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+    <title>Pet Connect</title>
+    <style>
+        section{
+            padding: 60px 0;
+        }
+    </style>
+</head>
+
+<body>
+    <!-- navbar -->
+    <nav class="navbar navbar-expand-md navbar-dark bg-primary"> 
+        <div class="container-xxl">
+            <a href="#intro" class="navbar-brand">
+                <span class="fw-bold text-white h2">
+                    Pet Connect
+                    <i class="bi bi-house-heart-fill text-white"></i>
+                </span>
+            </a>
+            <!-- toggle button for mobile nav -->
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+            data-bs-target="#main-nav" aria-controls="main-nav" aria-expanded="false"
+            aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <!-- navbar links -->
+            <div class="collapse navbar-collapse justify-content-end align-center" id="main-nav">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link text-white fw-bold" href="#ABOUT">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-white" href="#PET_LISTING_PAGE">List a Pet</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-white" href="#LIST_PET_FORM">Find a Pet</a>
+                    </li>
+                </ul>
+            </div>
         </div>
-      </div>
-    </header>
-  </body>
-</html>
+    </nav>
+
+    <!-- picture and intro -->
+    <section class="about bg-light">
+        <div class="container-lg row">
+            
+            <div class="col-8 justify-content-top">
```

Comment:
Try to avoid class names that contain presentational details like "col-8": https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming

If we want to change the widen the page, ideally we only need to change the CSS file and not the html as well.